package.json

```diff
@@ -87,9 +87,8 @@
     "mocha": "^2.4.5",
     "postcss-loader": "^1.1.1",
     "prismjs": "^1.5.1",
-    "react": "~15.3.0",
-    "react-addons-test-utils": "~15.3.2",
-    "react-dom": "~15.3.0",
+    "react": "^16.7.0",
```

Comment:
let's also add a peer dependency for `"react": ">=16.0.0 <17.0.0"` so that clients at least get a warning if they're not on react16. (even though we're not using any strict v16 features here yet, that'll make sure that we can if we choose)
e.g. https://github.com/Clever/goals-components/blob/master/package.json#L35